app/models/authenticator/httpd.rb

```diff
@@ -25,9 +29,15 @@ def user_authorizable_without_authentication?
       true
     end
 
-    def _authenticate(_username, _password, request)
-      request.present? &&
-        request.headers['X-REMOTE-USER'].present?
+    def _authenticate(username, password, request)
+      if !user_data_collected?(request) && request.present? && oidc_configured?
+        if password.present?
```

Comment:
don't we need to check that username is present instead ?

Reply:
Will do! Thank you.